File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
while True:
    arr = list(map(int, input().split()))
    arr.sort()

    a, b, c = arr
    if a == b == c:
        break
    if a ** 2 + b ** 2 == c ** 2:
        print('right')
    else:
        print('wrong')

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.cc
#include <bits/stdc++.h>

using namespace std;

int main()
{
    ios::sync_with_stdio(NULL);
    cin.tie(NULL);
    cout.tie(NULL);

    vector<int> arr(3);
    while (true)
    {
        cin >> arr[0] >> arr[1] >> arr[2];
        if (arr[0] == 0 && arr[1] == 0 && arr[2] == 0)
            break;
        sort(arr.begin(), arr.end());

        if (arr[0] * arr[0] + arr[1] * arr[1] == arr[2] * arr[2])
            cout << "right\n";
        else
            cout << "wrong\n";
    }
}